release/release_logs/2.2.0/scalability/single_node.json

```diff
@@ -1,8 +1,8 @@
 {
-    "args_time": 17.115523338999992,
-    "get_time": 26.32780160499999,
+    "args_time": 16.555890774999966,
+    "get_time": 25.818155864999994,
     "large_object_size": 107374182400,
-    "large_object_time": 255.3529769390001,
+    "large_object_time": 338.213855138,
```

Comment:
32% regression

Reply:
This is highly variable due to EBS https://github.com/ray-project/ray/issues/30711